### PR TITLE
docs: update docs with new proxy logging UI

### PR DESCRIPTION
**Pull request by flotwig**
for https://github.com/cypress-io/cypress/pull/16730

### CODE REVIEW

File: content/api/commands/intercept.md [hunk 1]
```diff
@@ -1537,6 +1537,50 @@ are some workarounds:
...
+## Command Log
+
+```javascript
+cy.intercept('/accounts*').as('accountsGet')
+cy.intercept('/company', { companyId: 1 }).as('companyGet')
+cy.intercept('/teams*', [{ teamId: 2 }]).as('teamsGet')
+```
+
+Whenever you create `cy.intercept()` rules, Cypress will display a new
+Instrument Log called _Routes_. It will list the routing table in the Instrument
```
Comment (cowboy) [inline content/api/commands/intercept.md:1549]: Elsewhere (see [The Test Runner](https://deploy-preview-4061--cypress-docs.netlify.app/guides/core-concepts/test-runner#Instrument-Panel) and [session](https://deploy-preview-4061--cypress-docs.netlify.app/api/commands/session#The-Instrument-Panel)) we called these Instrument Panels. Can we change that name throughout this PR?

File: content/api/commands/intercept.md [hunk 2]
```diff
@@ -1537,6 +1537,50 @@ are some workarounds:
...
+indicate whether they matched a `cy.intercept()` by the presence of a yellow
+badge on the right hand side:
+
+<DocsImage src="/img/api/intercept/command-log-fetches.png" alt="Screenshot of example fetches"></DocsImage>
+
+The circular indicator is filled if the request went to the destination server,
+but unfilled if the request was stubbed with a response.
+
+Clicking on a request that matched a `cy.intercept()` will print additional
+information about the request and response:
```
Comment (cowboy) [inline content/api/commands/intercept.md:1578]:
```suggestion
Clicking on a request that matched a `cy.intercept()` will print additional
information about the request and response to the console:
```

File: content/guides/guides/network-requests.md [hunk 3]
```diff
@@ -199,13 +199,13 @@ cy.intercept(
     url: '/users/*', // that have a URL that matches '/users/*'
   },
   [] // and force the response to be: []
-)
+).as('getUsers')
```
Comment (cowboy) [inline content/guides/guides/network-requests.md:202]: How about adding a comment, like:

```suggestion
).as('getUsers') // and assign an alias 
```

**Review by cowboy — CHANGES_REQUESTED** [review]
Overall, this looks great. It makes a lot of sense to me.

Question: what are the `<!-- ... -->` comments for? Are you going to remove them?

File: content/api/commands/intercept.md (hunk 1 above)
Comment (flotwig) [reply content/api/commands/intercept.md:1549]: copied the terminology from cy.route's docs, i'll update it in both places

File: content/api/commands/intercept.md (hunk 2 above)
Comment (flotwig) [reply content/api/commands/intercept.md]: b9f6578

File: content/guides/guides/network-requests.md (hunk 3 above)
Comment (flotwig) [reply content/guides/guides/network-requests.md]: 3c00591

**Comment by flotwig** [comment]
> Question: what are the `<!-- ... -->` comments for? Are you going to remove them?

Those comments are the source code that is used to generate the screenshots in the <DocsImage>s. Added comments to them in de87058 to indicate as suchh

File: content/api/commands/intercept.md (hunk 1 above)
Comment (flotwig) [reply content/api/commands/intercept.md]: de87058

**Review by cowboy — APPROVED** [review]
This all LGTM!

**Pull request merged**